doc/data/messages/c/consider-refactoring-into-while-condition/bad.py

```diff
@@ -0,0 +1,6 @@
+a = [1,2,3,4,5]
+
+while True:  # [consider-refactoring-into-while-condition]
+  if len(a) == 0:
+    break
+  a.pop()
```

Comment:
I like example that are less abstract and understandable by school children, it doesn't have to be fruit, it's just that I don't have a lot of imagination :)
```suggestion
fruit_basket = ["apple", "orange", "banana", "cherry", "guava"]

while True:  # [consider-refactoring-into-while-condition]
  if len(fruit_basket ) == 0:
    break
    
  fruit = fruit_basket.pop()
  print(f"We removed {fruit} from the basket")
```